draft-jholland-quic-multicast.md

```diff
@@ -151,15 +151,10 @@ multicast_client_params {
 ~~~
 {: #fig-transport-parameter-format title="multicast_client_params Format"}
 
-Capabilities Flags is a bit field structured as follows:
-
- - 0x1 is set if IPv4 channels are permitted
- - 0x2 is set if IPv6 channels are permitted
+The Capabilities, Max Aggregate Rate, and Max Channel ID fields are identical to their analogous fields in the MC_LIMITS frame ({{client-limits-frame}}) and hold the initial values.
```

Comment:
nit: here it is called "Capabilities Field" while in the frame definition it is called "Capabilities Flags"